test/find_2_conv.cpp

```diff
@@ -160,30 +160,79 @@ struct Find2Test : test_driver
 
         solutions.resize(100);
 
-        const auto search_values = std::vector<int>({0, 1});
-        const auto workspace_limit_values =
-            std::vector<std::size_t>({std::numeric_limits<std::size_t>::max(), 0});
-
-        for(const auto search : search_values)
-            for(const auto workspace_limit : workspace_limit_values)
-            {
-                miopenFindOptions_t options;
-
-                EXPECT_EQUAL(miopenCreateFindOptions(&options), miopenStatusSuccess);
-
-                EXPECT_EQUAL(miopenSetFindOptionTuning(options, search), miopenStatusSuccess);
-                EXPECT_EQUAL(miopenSetFindOptionResultsOrder(options, miopenFindResultsOrderByTime),
-                             miopenStatusSuccess);
-                EXPECT_EQUAL(miopenSetFindOptionWorkspaceLimit(options, workspace_limit),
-                             miopenStatusSuccess);
-
-                EXPECT_EQUAL(
-                    miopenFindSolutions(
-                        handle, problem, options, solutions.data(), &found, solutions.size()),
-                    miopenStatusSuccess);
-
-                EXPECT_EQUAL(miopenDestroyFindOptions(options), miopenStatusSuccess);
-            }
+        for(const auto search : {0, 1})
+            for(const auto workspace_limit : {std::numeric_limits<std::size_t>::max(), 0ul})
+                for(const auto prealloc : {true, false})
```

Comment:
I am afraid this is too much, as this doubles testing work (and time spent for CI). Maybe we can use pre-allocated buffers in the half of the test passes, e.g. in the even passes?

Reply:
This only runs for one config

Reply:
Good. Resolved.